ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -170,7 +172,7 @@ impl<P: Parameters> Add<Self> for GroupAffine<P> {
 impl<'a, P: Parameters> AddAssign<&'a Self> for GroupAffine<P> {
     fn add_assign(&mut self, other: &'a Self) {
         let mut s_proj = GroupProjective::from(*self);
-        s_proj.add_assign_mixed(other);
+        ProjectiveCurve::add_assign_mixed(&mut s_proj, other);
```

Comment:
Did this break? i.e. does `s_proj.add_assign_mixed(other)` no longer work?

Reply:
That's needed for disambiguation, since we have two traits that define that method